scripts/utils.js

```diff
@@ -176,6 +176,48 @@ async function updateDeploymentScriptVersion(nextVersion) {
   );
 }
 
+/**
+ * @param {string} importName
+ * @returns {[string, string]} [packageName, importPath]
+ */
+const getPackageNameFromImportName = (importName) => {
+  let [part1, part2, ...rest] = importName.split("/");
+  return importName.startsWith("@")
+      ? [`${part1}/${part2}`, rest.join("/")]
+      : [part1, [part2, ...rest].join("/")];
```

Comment:
I think even though we can't use the `limit` parameter here, the code still benefits from readability by hoisting the `startsWith("@")` logic:

```suggestion
  if (importSpecifier.startsWith("@")) {
    let [scope, pkg, ...path] = importSpecifier.split("/");
    return [`${scope}/${pkg}`, path.join("/")];
  }
  let [pkg, ...path] = importSpecifier.split("/");
  return [pkg, path.join("/")];
```